### ISSUE TITLE
Wont be able to uninstall/install the file if you manually deleted it.

### ISSUE THREAD

**Issue by MeizterRam** [Excrulon/Tree-of-Savior-Addon-Manager#14]
Steps to duplicate:
1. Install the files
2. Open and reopen TOS just to verify if the files are installed properly
3. Close TOS then open Steam/Steamapps/common/Tree of Savior/data
4. Delete the files that are installed by the addon manager.
You wont be able to install or uninstall the files

**Comment by Excrulon** [comment]
Yeah it's fixed in the latest commit. I'll release it eventually. People
shouldn't delete managed files manually though anyway...

**Comment by iurifabiooutlook** [comment]
I had to reinstall the game and now when I go to install addons it says already installer and does not uninstall. Steam / Steamapps / common / Tree of Savior / date does not have anything about the manager addons. I know to solve my problem?

**Comment by Halvaard** [comment]
* Starting over

If things ever get out of sync from some reason, you can find settings and addon cache in `%AppData%\Tree of Savior Addon Manager`. Open the run prompt or Windows Explorer and type that path in to open it.

There are two files here named `settings.json` and `addons.json`. `settings.json` is responsible for saving your Tree of Savior directory location. `addons.json` is responsible for keeping track of the addons you have installed. If something goes wrong and things get out of sync, either manually make the changes in this file or delete that file and the installed ipfs manually and start over. Hopefully you won't really have to do this.
